plugins/dfu/dfu-device.h

```diff
@@ -46,7 +45,6 @@ typedef enum {
 	DFU_DEVICE_QUIRK_NO_GET_STATUS_UPLOAD	= (1 << 5),
 	DFU_DEVICE_QUIRK_NO_DFU_RUNTIME		= (1 << 6),
 	DFU_DEVICE_QUIRK_ATTACH_UPLOAD_DOWNLOAD	= (1 << 7),
-	DFU_DEVICE_QUIRK_IGNORE_RUNTIME		= (1 << 8),
```

Comment:
As a result of pulling out "one in the middle" there is a hole.  Since this is "internal", shuffle them all down?